### PR TITLE
CORE-2006: updated dependencies and fixed some lint warnings

### PR DESCRIPTION
**Pull request by slr71**

### CODE REVIEW

File: src/common_cli/version.clj [hunk 1]
```diff
@@ -10,11 +11,7 @@
 
 (defn jar-file-stream
   [path-in-jar]
-  (.openStream
-   (java.net.URL.
-    (str "jar:"
-         (jar-domain)
-         path-in-jar))))
```
Comment (ianmcorvidae) [inline src/common_cli/version.clj:17]: Does this work the same? The old one appears to return an open stream, where the latter looks to only return the URI object. I'm not even sure where we use this function, but nonetheless thought I should ask.

**Review by ianmcorvidae — APPROVED** [review]
Had one question, basically looks fine to me though other than that.

File: src/common_cli/version.clj (hunk 1 above)
Comment (slr71) [reply src/common_cli/version.clj:17]: Ah, good catch. I think I got a little careless there. I'll check into it and fix it.

Comment (slr71) [reply src/common_cli/version.clj:17]: I think it should be fixed now.

**Comment by slr71** [comment]
I'm going to merge these changes now. Thanks for the review! 👍

**Pull request merged**